jbpm-designer-backend/src/test/java/org/jbpm/designer/web/server/TransformerServletTest.java

```diff
@@ -372,9 +374,11 @@ public void testStoreInRepository() throws Exception {
                                        new FilterByExtension(targetType));
         assertEquals(1,
                      assets.size());
-        asset = repository.loadAsset(assets.iterator().next().getUniqueId());
-        assertNotNull(asset);
-        assertNotNull(asset.getAssetContent());
+        Asset<String> updatedAsset = repository.loadAsset(assets.iterator().next().getUniqueId());
+        assertNotNull(updatedAsset);
+        assertNotNull(updatedAsset.getAssetContent());
+        // make sure the asset has the new content
+        assertTrue(new String(updatedAsset.getAssetContent()).contains("id=\"_modifiedFormattedSVG\""));
```

Comment:
Optional nitpick: `getAssetContent()` returns `String` here, so there should be no need to wrap it in String constructor. Also you could get better error message if you replaced the comment with assert message:
`assertTrue("Asset should contain the new content", updatedAsset.getAssetContent().contains("id=\"_modifiedFormattedSVG\""));`